src/range.ts

```diff
@@ -0,0 +1,11 @@
+export function range(start: number, end: number, step = 1): Array<number> {
```

Comment:
I think all exported "public api" members should have TSDoc (or any doc at all really)